### PR TITLE
Change the pacakge dependency to point to the "China" compdata branch.

### PR DESCRIPTION
**Pull request by pralitp**
**NOTE**: we won't want this change when we finally merge to master.

### CODE REVIEW

**Comment by pralitp** [comment]
We will want to pay close attention to the travis / appveyor logs to make sure they do indeed pull the right branch for the compdata.

**Comment by rynacui** [comment]
Noted! 
As we push more comparison files to the China compdata branch, are they going to that referenced PR automatically?

**Comment by pralitp** [comment]
That is the hope!

**Comment by pralitp** [comment]
From the appveyor log: `Downloading GitHub repo JGCRI/gcamdata.compdata@gcamdata-China` that looks good!

But from travis: `Downloading GitHub repo jgcri/gcamdata.compdata@HEAD`.  Not good 👎 .  I think I know the problem however.  I'll push another commit..

**Comment by pralitp** [comment]
Ok, not super clear in the Travis log but it does note: `Github (JGCRI/gcamdata.compdata@df1fa84)`

Which corresponds to the last commit @rynacui made so I think we are all set!

@rynacui, I think Ben may be on vacation so we can likely move forward on this PR without him if you think it looks good.

File: DESCRIPTION [hunk 1]
```diff
@@ -49,7 +49,7 @@ Suggests:
     testthat (>= 1.0.2),
     R.utils (>= 2.6.0),
     gcamdata.compdata
-Remotes: github::JGCRI/gcamdata.compdata
+Remotes: github::JGCRI/gcamdata.compdata#16
```
Comment (bpbond) [inline DESCRIPTION]: Huh, interesting.

**Comment by bpbond** [comment]
FYI - no I'm not on vacation until the 20th.

**Pull request merged**